### DISCUSSION TITLE
How to setup webhook to enable Github pull request status check?

### DISCUSSION THREAD

**Discussion by tchiaspko** [sorry-cypress/sorry-cypress#436] [Q&A] [answered]
Does anyone have some sample values to setup the webhook to show when the cypress is running and completed and the final status (pass/fail) under the Pull Request Check section? 

There seems to be two types of webhook. Generic and Github. which one should I use? 
 
Current when I tries to setup Generic Webhook, it requires the following fields
URL
Headers
Events:

Can someone give a sample URL for a particular github repo? 
What are some example Headers to specify to login to github? are there anything neede besides the login credentials?

**Comment by agoldis** [comment, marked as answer]
<img width="1449" alt="Screen Shot 2021-09-10 at 10 18 13 AM" src="https://user-images.githubusercontent.com/1637928/132892500-7862f7f6-d715-444a-9f0b-8d8846554120.png">

Repo URL: https://github.com/sorry-cypress/sorry-cypress

Github token:
<img width="1010" alt="Screen Shot 2021-09-10 at 10 19 15 AM" src="https://user-images.githubusercontent.com/1637928/132892614-b3eea914-7916-4cec-831b-263a8c68aa95.png">

**Reply by tchiaspko** [reply]
Thanks. This update the PR at the end of the execution.
